operations/_objects.md

```diff
@@ -199,4 +199,27 @@ Usage of this value is **deprecated**. Where possible, use the properties exposi
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Currency` | string | required |  |
-| `Value` | number | required |  |
\ No newline at end of file
+| `Value` | number | required |  |
+
+#### Resource Feature Classification
```

Comment:
FYI: moved to common objects since it is used in `Resource Features` and `Customers` endpoints